libs/debugging/include/hpx/debugging/print.hpp

```diff
@@ -61,6 +64,11 @@
 // The output will only be produced every N seconds
 // ------------------------------------------------------------
 
+// Used to wrap function call parameters to prevent evaluation
+// when debugging is disabled
+#define DP_LAZY(Expr, printer) \
```

Comment:
Please prefix the macro name with `HPX_` to avoid unwanted name-clashes with macro names defined in other places.

Reply:
done